ml-agents/mlagents/trainers/model_saver/torch_model_saver.py

```diff
@@ -75,16 +76,20 @@ def initialize_or_load(self, policy: Optional[TorchPolicy] = None) -> None:
             )
         elif self.load:
             logger.info(f"Resuming from {self.model_path}.")
-            self._load_model(self.model_path, policy, reset_global_steps=reset_steps)
+            self._load_model(
+                os.path.join(self.model_path, "checkpoint.pt"),
+                policy,
+                reset_global_steps=reset_steps,
+            )
 
     def _load_model(
         self,
         load_path: str,
         policy: Optional[TorchPolicy] = None,
         reset_global_steps: bool = False,
     ) -> None:
-        model_path = os.path.join(load_path, "checkpoint.pt")
-        saved_state_dict = torch.load(model_path)
+        print("loading: ", load_path)
+        saved_state_dict = torch.load(load_path)
```

Comment:
load_path is always the full path to the .pt file. It originally comes from trainer_settings.init_path which is assigned in learn.py